drive/modules/listener/http_service.bal

```diff
@@ -101,5 +104,17 @@ service class HttpService {
             check caller->respond(http:STATUS_OK);
         }
     }
+    resource isolated function get [string name](http:Caller caller) returns @tainted error? {
+        http:Response r = new();
```

Comment:
variable name with single letter is not a good convention.

Reply:
Yeah. Will change this.